spark/src/test/java/org/apache/zeppelin/spark/DepInterpreterTest.java

```diff
@@ -74,20 +73,6 @@ public void setUp() throws Exception {
   @After
   public void tearDown() throws Exception {
     dep.close();
-    delete(tmpDir);
```

Comment:
@DrIgor Thanks for the contribution.
is there a reason not reamoving tmpDir after the class?